src/20_Components/amp-live-list.html

```diff
@@ -0,0 +1,147 @@
+  When the number is equal to 10, the server stops generating new content. If you want to see the demo again, delete the cookie or access the page using an Incognito tab.
+
+-->
+<!-- -->
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <!-- #### Setup -->
+  <!--
+  Import the `amp-live-list` component in the header.
+
+  Use `amp-live-list-item` to style the children of the `amp-live-list`. Refer to the [documentation](https://github.com/ampproject/amphtml/blob/f2981e72843c8e71a9cd2535282e4c4947bb6882/extensions/amp-live-list/amp-live-list.md) for all the details about styling.
+  -->
+  <script async custom-element="amp-live-list" src="https://cdn.ampproject.org/v0/amp-live-list-0.1.js"></script>
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+  
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+  <style amp-custom>
+    amp-live-list > [items] {
+      display: flex;
+      flex-direction: column;
+    }
+
+    .amp-live-list-item {
+      background-color: #f8f8f8;
+      border-bottom: 1px solid rgba(0, 0, 0, .12);
+    }
+
+    .card-list {
+      display: flex;
+      margin: 15px;
+      padding: 10px;
+      align-items: center;
+      background-color: #fff;
+      box-shadow: rgba(0, 0, 0, .3) 0 0 5px;
+    }
+
+    .content {
+      font-size: 1.5rem;
+      margin-left: 15px;
+    }
+
+    .button {
+      border: none;
+      border-radius: 2px;
+      color: #fafafa;
+      height: 26px;
+      min-width: 32px;
+      padding: 0 16px;
+      margin: 0 16px;
+      text-transform: uppercase;
+      letter-spacing: 0;
+      cursor: pointer;
+      vertical-align: middle;
+      line-height: 26px;
+      text-align: center;
+      background: #3f51b5;
+    }
+
+    header {
+      font-size: 2rem;
+    }
+  </style>
+</head>
+<body>
+
+  <div>
+    <h3>Live updates</h3>
+    <h4>Refresh takes 15 seconds.</h4>
+  </div>
+  <!-- #### amp-live-list with insert -->
+  <!-- 
+    Use `data-poll-interval` attribute to specify how frequently the `amp-live-list` component will poll new data. Values are valid above the minimum of 15 seconds. 
```

Comment:
s/Values are valid above the minimum of 15 seconds./The minimum interval is 15 seconds./